### DISCUSSION TITLE
Using the builder for Input, Submit elements, etc. not SchemaField

### DISCUSSION THREAD

**Discussion by ile** [alibaba/formily#3772] [Q&A 2.x]
(I'm new here, so I'm asking some newbie questions.)

I'm trying to get the builder into use, but it creates a form that has `<SchemaField>`, `<SchemaField.String>` etc. elements.

But I would like to use the elements from `@formily/antd` (or `@formily/antd-v5`) directly, like the example does here: https://formilyjs.org/guide/quick-start

```javascript
import {
  FormItem,
  FormLayout,
  Input,
  FormButtonGroup,
  Submit,
} from '@formily/antd'
```

Is it possible to use the builder to create these `antd` elements? It would be much easier to edit the created source code if it consists of `antd` elements.

Thank you.

**Comment by janryWang** [comment]
I understand it is possible, but this build tool needs to be done by yourself, because it will involve AST parsing, analyze component dependencies, and then do automatic loading
